README.md

```diff
@@ -14,10 +14,11 @@
 
 
 ### I get it, my data might be lost, is that a problem?
-   
-* What's the repercussion of the failure?
-* It depends on the business case
-* Some scenarios like charging credit cards cause customers to be reluctant to "try again" being afraid to be double charged
+
+As always, the correct answer is "it depends". If you are building a new social media site you might, at least initially, be OK with losing user's updates from time to time. Some users might be annoyed by this but no big deal.
+
+If, on the other hand, you are processing credit card payments and your site stops responding in the middle of the payment process users might be a little bit reluctant to just open a new browser tab and try again. Everybody loves Porsche but having two of them delivered to your driveway might a bit too much for your bank account.
```

Comment:
Do we want to elaborate on the flip side? Data is saved but the client doesn't know leaving the customer frustrated, afraid to submit again, frantically hitting refresh in gmail to see if there is a confirmation email coming?

I actually hit this last friday, decided to hold of with the purchase and then the email arrived on Sunday 😀 

It fair to say that I wasn't impressed

Reply:
Isn't this the an instance of the same issue as with CC processing and not wanting to buy two Porsches?

Reply:
Ok, I see it now 👍